### ISSUE TITLE
[2021 fall] CVPR 2021 VinVL: Revisiting Visual Representations in Vision-Language Models (20208147)

### ISSUE THREAD

**Issue by psh01087** [awesome-davian/awesome-reviews-kaist#141]

**Comment by quangbk2010** [comment]
Hello, this is Quang.
It's my pleasure to review your paper review.

Overall, I think it's a clear review and easy to follow.
There are some minor adjustment maybe considered.

1. The 1st equation, it should be separated a little bit more to make it less confused between (q,v) and y.
2. 'The main difference between Oscar and Oscar comes from the pre-training loss', may be a typo.
3. Some reference are highlighted as bold text, but some are not.
4. The experiment settings maybe necessary to add, such as: which datasets (I see they are described in section3, but they should be in the result section for easier following), ...
5. The tables contain 2 descriptions.
6. Table 2 should be in the Result section.

If there is any misunderstanding, please correct me.
Thank you.

**Comment by sateodoro** [comment]
Hello! I am Samuel Teodoro (20218258).

In terms of content, the review is well-written. Just a small formatting issue:
- The first equation is quite confusing because of the spacing. It would be better if there is a space or new line.

**Comment by pranjay-unof** [comment]
Hello, I am Pranjay Shyam (20185586) the reviewer assigned for this paper review. 

I would like to summarize the areas of revision as,

- In the paper equation 1 consists of 2 parts. Rewriting it as two separate equations would improve the readability. 
- The caption for Tables and Figures appears twice.
- I would encourage a consistent font, i.e. not to highlight the references.
- The paper contains extended ablation evaluations demonstrating the effect of different components. I would encourage them to be considered in the paper such as  "Effect of self-training" and "Effect of the new 3-way contrastive loss". I believe these to be more important. 
- There are some typos in the paper that can be corrected. 

The overall content of the review is good and easy to follow.
